release/_releases_json.sh

```diff
@@ -52,7 +52,10 @@ function _process_product {
 		tr -d "/" | \
 		uniq)
 	do
-		_process_product_version "${product_name}" "${product_version}"
+		if [[ ! "{$product_version}" =~ (7\.[0-9]+\.[0-9]+\-u1(1[3-9]|[2-9][0-9]+|[0-9]{3,})) ]]
```

Comment:
Hey @pgydev,

I just realized that the variable in the if should be like `${product_version}` instead of `{$product_version}`, am I right?

Reply:
Hi, @natocesarrego ,

Yes i corrected it. I hope i didn't do any more typo :sweat_smile: